src/test/java/ByeCommandTest.java

```diff
@@ -0,0 +1,12 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+class ByeCommandTest {
+
+    @Test
+    void isExitTest() {
```

Comment:
Hmm maybe you can consider naming your test methods with the following format `featureUnderTest_testScenario_expectedBehavior()`

Reply:
Do you mean that the position of the name can be replaced by a better description? Could you give an example, please?

Reply:
I meant that for test method names it may be better to describe the test within your method name. One example given is `intDivision_zeroDivisor_exceptionThrown`, which describes to readers that the test is testing for integer division using zero as a divisor, and the expected result is an exception being thrown. 

In this case maybe you can consider naming it `isExitTest_byeCommand_returnsTrue`? Again this is just a suggestion though, apparently the coding standard guide states  that the "Third part or both second and third parts can be omitted depending on what's covered in the test."

Reply:
Clear example given! Thank you so much.